### PR TITLE
DoxyGen: Add missing device properties to condition.

### PR DESCRIPTION
**Pull request by JonatanAntoni**

### CODE REVIEW

**Review by jkrech — CHANGES_REQUESTED** [review]
Please check my comment regarding wrong symbol.

File: doxygen/src/conditions_schema.txt [hunk 1]
```diff
@@ -222,43 +222,55 @@ A \ref element_condition "condition" becomes \token{true} when:
...
     <td>Ddsp</td>
     <td>Specifies whether Digital Signal Processing (DSP) instruction set must be supported by the device or not. Predefined values can be uses as listed in the table \ref DdspEnum "Device DSP".</td>
-    <td>DdspEnum</td>
+    <td>\ref DdspEnum</td>
     <td>optional</td>
   </tr>
   <tr>
+    <td>Dmve</td>
+    <td>Specifies whether the M-Profile Vector instruction set extension must be supported by the device or not. Predefined values can be used as listed in the table \ref DmveEnum "Device MVE".</td>
+    <td>\ref DdmveEnum</td>
```
[the remark this answers is not in the document]
Comment (JonatanAntoni) [reply doxygen/src/conditions_schema.txt:261]: Good catch. Such issues should be pointed out by CI, already. Turned out this check was missing.

[the remark this answers is not in the document]
Comment (JonatanAntoni) [reply doxygen/src/conditions_schema.txt:261]: Now the doxygen warnings are shown as expected.

**Review by jkrech — APPROVED** [review]
LGTM

**Pull request merged**